pkg/collector/dist/conf.d/docker.yaml.example

```diff
@@ -42,6 +42,11 @@ instances:
     #
     # collect_image_size: true
 
+    # Monitor containers exit codes and send critical service checks when exit code is not 0
```

Comment:
it also sends OK service checks when the exit code is 0 right? If so we should write it as well